packages/mux-player/src/template.ts

```diff
@@ -14,16 +14,54 @@ import { i18n, stylePropsToString } from './utils';
 import type { MuxTemplateProps } from './types';
 import { StreamTypes } from '@mux/playback-core';
 
+const controlsListStyles: Record<string, string> = {
+  notop: `::part(top), [part~="top"] { display: none; }`,
+  nobottom: `::part(bottom), [part~="bottom"] { display: none; }`,
+  nocenter: `::part(center), [part~="center"] { display: none; }`,
+
+  nocenterplay: `::part(center play button) { display: none; }`,
+  nocenterseekbackward: `::part(center seek-backward button) { display: none; }`,
+  nocenterseekforward: `::part(center seek-forward button) { display: none; }`,
+
+  noplay: `::part(bottom play button) { display: none; }`,
+  noseekbackward: `::part(bottom seek-backward button) { display: none; }`,
+  noseekforward: `::part(bottom seek-forward button) { display: none; }`,
+
+  nomute: `::part(mute button) { display: none; }`,
+  nocaptions: `::part(captions button) { display: none; }`,
+  noairplay: `::part(airplay button) { display: none; }`,
+  nopip: `::part(pip button) { display: none; }`,
+  nocast: `::part(cast button) { display: none; }`,
+  nofullscreen: `::part(fullscreen button) { display: none; }`,
+  noplaybackrate: `::part(playbackrate button) { display: none; }`,
+  novolumerange: `::part(volume range) { display: none; }`,
+  notimerange: `::part(time range) { display: none; }`,
+  notimedisplay: `::part(time display) { display: none; }`,
+  noremoteplayback: `::part(airplay button), ::part(cast button) { display: none; }`,
+
+  // the seek live button is in the light dom below, use attribute selector.
+  noseeklive: `[part*="seek-live button"] { display: none; }`,
+};
+
 export const template = (props: MuxTemplateProps) => html`
   <style>
     ${cssStr}
+    ${[...props.controlsList]
+      .map((token) => controlsListStyles[token])
+      .filter(Boolean)
+      .join('\n')}
   </style>
   ${content(props)}
 `;
 
 const flatCSSParts = Object.values(parts).flatMap((group) => group.split(' '));
 const forwardUniqueCSSParts = [...new Set(flatCSSParts)].join(', ');
 
+const isLive = (props: MuxTemplateProps) => [StreamTypes.LIVE, StreamTypes.LL_LIVE].includes(props.streamType as any);
+
+const isLiveOrDVR = (props: MuxTemplateProps) =>
```

Comment:
suggestion(non-blocking): If we're going to use these functions, we should probably defer the "live" half to `isLive` here. i.e.:

```js
isLive(props) || [StreamTypes.DVR, StreamTypes.LL_DVR].includes(props.streamType as any);
```